src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(this.getName() + " start work !");
```

Comment:
```suggestion
        System.out.println("Bulldozer start work !");
```

Let's explicitly write the names for this task